.github/workflows/main.yml

```diff
@@ -46,4 +46,4 @@ jobs:
       # Awk One-liner stolen from https://stackoverflow.com/questions/27447705/grep-without-filtering 
 
     - name: Git Clean Check
-      run: test -z "$(git status --porcelain)" || { git diff; exit 1; }
+      run: test -z "$(git -c core.fileMode=false status --porcelain)" || { git diff; exit 1; }
```

Comment:
```chmod +x gradlew``` on line 34 is causing the git status to not be clean. The added option ignores permission differences to get around this. 
I'm not sure why it became necessary when we didn't need it previously in 2022RobotCode. Maybe it was set globally for that repo?

Reply:
The file was chmod'd and committed in 2022.

The chmod likely wasn't needed on line 34 in 2022, and therefore didn't cause porcelain to fail.
in 2023, gradlew is not executable...so the chmod is needed, and causes porcelain to fail

casey@obsidian:~/Documents/code$ ls -l 202*RobotCode/gradlew
-rwxrwxr-x 1 casey casey 8047 Aug 24 08:05 2022RobotCode/gradlew
-rw-rw-r-- 1 casey casey 8047 Aug 31 16:31 2023RobotCode/gradlew